packages/server-api/src/index.ts

```diff
@@ -87,3 +89,75 @@ export interface Plugin {
   signalKApiRoutes?: (router: IRouter) => IRouter
   enabledByDefault?: boolean
 }
+
+export type DeltaInputHandler = (
+  delta: object,
+  next: (delta: object) => void
+) => void
+
+export interface Ports {
+  byId: string[]
+  byPath: string[]
+  byOpenPlotter: string[]
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  serialports: any
+}
+
+export interface ServerAPI extends PluginServerApp {
+  getSelfPath: (path: string) => void
+  getPath: (path: string) => void
+  getMetadata: (path: string) => void
```

Comment:
These definitions do not have any return type.

Reply:
Merged already, start a new one please.